modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/ParameterModelsLoaderDelegate.java

```diff
@@ -254,6 +256,12 @@ private void parseConfigOverride(ExtensionParameter extensionParameter, Paramete
 
     declarer.withDslInlineRepresentation(groupAnnotation.showInDsl());
 
+    DisplayName displayName = groupParameter.getAnnotation(DisplayName.class).orElse(null);
+
+    if (displayName != null) {
```

Comment:
This can be replaced with `groupParameter.getAnnotation(xxxxxx.class).ifPresent(displayName -> <logic-to-build-display-model>);`